src/compatibilityApi/java/com/gradle/ccud/adapters/DevelocityAdapter.java

```diff
@@ -0,0 +1,39 @@
+package com.gradle.ccud.adapters;
+
+import org.gradle.api.Action;
+import org.gradle.caching.configuration.AbstractBuildCache;
+
+import javax.annotation.Nullable;
+
+/**
+ * Adapter for {@link com.gradle.develocity.agent.gradle.DevelocityConfiguration} and {@link com.gradle.ccud.adapters.enterprise.proxies.GradleEnterpriseExtensionProxy}
+ */
+public interface DevelocityAdapter {
```

Comment:
Is this DevelocityApiAdapter in the Maven extension?

Reply:
This is the `CoreApiAdapter`. Finding a suitable name for the root adapter is a bit tricky :)